site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,287 @@
+| Requester | An individual who establishes an annotation project within CVAT determines the price, creates tasks and jobs within the project, and then releases it on Toloka.                                                                                        |
+| Toloker   | A person who annotates the Requester's dataset.                                                                                                                                                                                                         |
+
+<!--lint enable maximum-line-length-->
+
+## Preconditions
+
+Actor: Requester.
+
+Requester must have a [CVAT account](/docs/manual/basics/registration/)
+and [Toloka Requester account](https://platform.toloka.ai/auth?role=REQUESTER&retpath=%2Fsignup%2Frequester).
+
+To get access to the feature in CVAT, send request to [CVAT Support](mailto:support@cvat.ai)
+
+## Creating Toloka project
+
+The requester can set up a project within CVAT and subsequently
+connect it to Toloka, making it accessible for annotations by Tolokers.
+
+To initiate your Toloka project, proceed with the following steps:
+
+1. [Log in to CVAT](/docs/manual/basics/registration/#account-access)
+   and [initiate a new Project](/docs/manual/advanced/projects/).
+   <br>Here you can can setup user guide which will be shown on Toloka platform,
+   see [Adding instructions for annotators to the Toloka project](#adding-instructions-for-annotators-to-the-toloka-project).
+2. Navigate to the project page and select **Actions** > **Setup crowdsourcing project**.
+
+   ![Setting Up Toloka Project](/images/toloka01.png)
+
+3. Fill in the following fields in the **Setup crowdsourcing project** form:
+
+   ![Toloka Project Configuration](/images/toloka02.png)
+
+   - **Provider**: Choose Toloka as your provider.
+   - **Environment**: Select either **Sandbox** for a testing environment or **Production** for a live environment.
+   - **API key**: Enter the [Requester API key](https://toloka.ai/docs/toloka-kit/registration/).
+   - **Project description** (Optional): Provide a brief description of your project.
+
+4. Click **Submit**.
+   A pop-up indicating **Toloka Project Created** will appear, along with the **Update project** form.
+
+   ![Project Update Form](/images/toloka03.png)
+
+   - ![Open Project in Toloka](/images/toloka04.png) will take you to the published project in Toloka.
+   - ![Open Project in CVAT](/images/toloka05.png) will take you back to the project in CVAT.
+
+In CVAT, all projects related to Toloka will be labeled as **Toloka**.
+
+![Toloka Label on Project](/images/toloka06.png)
+
+The status indicator changes based on the
+state of the project and pool.
+
+For projects:
+
+- Green - active
+- Dark grey - archived
+
+For pools:
+
+- Green - open for annotating
+- Light gray - closed
+- Dark grey - archived
```

Comment:
Why we cannot add it in the setup Toloka pool section?

Reply:
moved to the pools section